### PR TITLE
#PAB-11 - create ingester endpoint

### PR DESCRIPTION
**Pull request by cyrusdobbs**
### Link to ticket
[#PAB-11](https://trello.com/c/4GcJ8Tmm/11-build-an-ingester-to-take-data-from-the-excel-spreadsheet)

### Change description
Adds the /ingest end point responsible for ingesting an Excel sheet and transforming it to JSON format

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Use Swagger UI or any REST Client to test the endpoint.
There is a test file in tests/resources but you can any excel file (e.g. the data model excel file)

### CODE REVIEW

File: controllers/ingest.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    :return: Excel sheet in json format
+    """
+    excel_file = connexion.request.files["excel_file"]  # required
+    sheet_names = body.get("sheet_names", None)
+
+    if sheet_names == [""]:
+        sheet_names = None
+
+    if excel_file.content_type != EXCEL_MIMETYPE:
+        return bad_request_error("Invalid file type"), 400
```
Comment (gidsg) [inline controllers/ingest.py:63]: Could we use the flask `abort` helper instead? e.g. https://flask.palletsprojects.com/en/2.2.x/errorhandling/?highlight=abort#custom-error-pages

Comment (cyrusdobbs) [reply controllers/ingest.py]: ah great suggestion - just what I was looking for

File: requirements.in [hunk 2]
```diff
@@ -12,3 +12,11 @@ Flask==2.2.3
...
 #-----------------------------------
 connexion[swagger-ui]==2.14.1
+prance==0.22.11.4.0
+openapi-spec-validator==0.5.5
+
+#-----------------------------------
+#   Excel Ingestion
+#-----------------------------------
+pandas==1.5.3
+openpyxl==3.1.1
```
[the remark this answers is not in the document]
Comment (cyrusdobbs) [reply requirements.in]: 👍

**Review by gidsg — APPROVED** [review]
LGTM 🐚 👍

**Pull request merged**